dfusion_rust_core/src/models/account_state.rs

```diff
@@ -51,6 +51,14 @@ impl AccountState {
         self.state_index = self.state_index.saturating_add(U256::one());
         self.state_hash = self.rolling_hash(self.state_index.low_u32());
     }
+
+    pub fn apply_withdraws(&mut self, withdraws: &[PendingFlux]) {
```

Comment:
Ideally this logic is moved from wherever it already exists in the driver (potentially a separate PR which has to come first).

Reply:
I am pretty sure this is the appropriate place in the code for such a function (considering apply deposits is right here in the same file).

Reply:
I agree, my point is it must already exist somewhere else today. And my ask would be to move the logic from there into here (cf. #192)

Reply:
Ahh! Yes! I see. Interesting! I will find it and deal with that!